app/views/posts/index.html.erb

```diff
@@ -2,7 +2,7 @@
 
   <%= image_tag post.image_url.to_s %>
   <%= post_presenter(post) %>
-  
+
```

Comment:
I'd watch these sorts of changes as they can lead to churn and needless merge conflicts. Try `git add -p` to selectively select chunks to commit.